meteor/server/api/peripheralDevice.ts

```diff
@@ -256,12 +257,19 @@ export namespace ServerPeripheralDeviceAPI {
 							PlayoutLockFunctionPriority.CALLBACK_PLAYOUT,
 							async () => {
 								const rundownIDs = Rundowns.find({ playlistId }).map((r) => r._id)
```

Comment:
If you want to optimise further, you could make this query load only the `_id` field and not the whole document